style.css

```diff
@@ -430,3 +414,203 @@ header {
 #form .btn:active {
   background-color: #2230d2;
 }
+
+/* Media Queries */
+@media (max-width: 400px) {
+  .cards {
+    padding: 16px;
+  }
+}
+
+@media (max-width: 767px) {
+  .nav-links {
+    display: none;
+  }
+}
+
+@media (min-width: 768px) {
+  header {
+    padding: 1.5% 9%;
+  }
+
+  header .fa-bars {
+    display: none;
+  }
+
+  header #logo {
+    display: flex;
+    align-items: center;
+  }
+
+  header nav ul {
+    list-style-type: none;
+    display: flex;
+  }
+
+  header nav ul li {
+    padding: 10px 12px;
+  }
+
+  header nav a {
+    text-decoration: none;
+    font-size: 15px;
+    color: #344563;
+    font-weight: 500;
+    line-height: 20px;
+    cursor: pointer;
+  }
+
+  header nav button {
+    display: none;
+  }
+
+  .main-content {
+    background-image: url(./asset/header-bg-desktop.svg);
+    background-repeat: no-repeat;
+    background-size: 97%;
+    background-position: 100% 10%;
+    justify-content: center;
+  }
+
+  .main-content-container {
+    width: 70%;
+  }
+
+  /* Card Section */
+
+  .card-container {
+    padding: 9%;
+  }
+
+  .cards {
+    flex-direction: row;
+    gap: 10px;
+  }
+
+  .cards > div {
+    width: 50%;
+  }
+
+  .cards .screenshot {
+    object-fit: cover;
+    height: 100%;
+  }
+
+  .card-description {
+    padding: 1% 2%;
+  }
+
+  .card-2,
+  .card-4 {
+    flex-direction: row-reverse;
+  }
+
+  /* About Section */
+  .about-container {
+    grid-template-columns: 45% 55%;
+    padding: 9%;
+    column-gap: 15px;
+  }
+
+  .skill-container {
+    padding-top: 0;
+  }
+
+  .skill-item:nth-child(1) {
+    padding: 8px 12px 24px 12px;
+  }
+
+  .skill-item:nth-child(2),
+  .skill-item:nth-child(3) {
+    padding: 24px 12px;
+  }
+
+  .language-container {
+    display: flex;
+    flex-direction: column;
+  }
+
+  .language-list {
+    flex-direction: row;
+    gap: 12px;
+    margin-left: 12px;
+  }
+
+  .language-list li div {
+    width: 115px;
+    height: 100px;
+    height: 100%;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-around;
+    background-color: #f7f7f9;
+    padding-top: 12px;
+    padding-left: 12px;
+  }
+
+  .language-item img {
+    width: 48px;
+  }
+
+  .language-item p {
+    margin-left: 0;
+    margin-top: 10px;
+  }
+
+  /* Contact Form */
+
+  #contact-container {
```

Comment:
- [ ] There are some slight differences in the contact form between different devices. Please make sure the button is aligned in the center and the inputs have sharp corners on the desktop version. 🧐️

**Your design**
![Screenshot from 2023-05-04 18-12-48](https://user-images.githubusercontent.com/17243325/236256109-97458f9f-bbe1-40e1-b545-8ee994865f26.png)

**Figma design**
![Screenshot from 2023-05-04 18-11-43](https://user-images.githubusercontent.com/17243325/236256253-ae19ca79-46b9-46ed-9c46-a07f1904aff4.png)